egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -358,6 +358,9 @@ def train(args, run_opts):
                                right_context_final >= 0 else -1)
 
     default_egs_dir = '{0}/egs'.format(args.dir)
+    if args.egs_dir is not None:
+        logger.warning("the --feat.cmvn-opts option has no effect because we are not dumping egs")
```

Comment:
Can you change this to just warn if those respective options are actually set?  Would need two more if statements inside this block.

Reply:
I have updated the pull request, may you take a look when you are available, thank you!